src/libs/ReportUtils.ts

```diff
@@ -2904,6 +2912,26 @@ function buildOptimisticIOUReport(payeeAccountID: number, payerAccountID: number
     };
 }
 
+/**
+ * Populates the report field formula with the values from the report and policy.
+ * Currently, is supports only optimistic expense reports.
+ * Each formula field is either replaced with a value, or removed.
+ * If after all replacements the formula is empty, the original formula is returned.
+ */
+function populateOptimisticReportFormula(formula: string, report: OptimisticExpenseReport, policy: Policy | EmptyObject): string {
```

Comment:
There's a couple more we can handle here optimistically

`{report:created}`
`{report:created:yyyy-MM-dd}`
`{report:status}`
`{user:email}`
`{user:email|frontPart} would output bob assuming that is the currently logged in Expensify user’s email.`